public/app/features/admin/AdminEditOrgPage.tsx

```diff
@@ -29,6 +29,17 @@ const getOrgUsers = async (orgId: UrlQueryValue, page: number) => {
   return { orgUsers: [] };
 };
 
+const getUsersRoles = async (orgId: number, users: OrgUser[]) => {
+  const userIds = users.map((u: OrgUser) => u.userId);
+  const roles = await getBackendSrv().get(
+    `/api/access-control/users/roles`,
+    accessControlQueryParam({ userIds, targetOrgId: orgId })
+  );
+  users.forEach((u: OrgUser) => {
+    u.roles = roles ? roles[u.userId] || [] : [];
+  });
+};
```

Comment:
```suggestion
const getUsersRoles = async (orgId: number, users: OrgUser[]) => {
  const userIds = users.map((u) => u.userId);
  const roles = await getBackendSrv().get(
    `/api/access-control/users/roles`,
    accessControlQueryParam({ userIds, targetOrgId: orgId })
  );
  users.forEach((u) => {
    u.roles = roles ? roles[u.userId] || [] : [];
  });
};
```
The OrgUser type here can be omitted since it's inferred from the `users`.